Submodels/Legacy/testing_averaging.py

```
@@ -0,0 +1,69 @@
+import re
+import glob
+
+# Construct output file name
+output_file = f'averaged_file.test_case'
+
+skip_lines: dict = {2, 14, 26, 27, 32, 37, 42}
+
+# Construct a simple local function to load in the contents of each file in the directory
+def read_file(filename):
```

Ruff (0.8.2)

Missing return type annotation for public function `read_file`

(ANN201)